### ISSUE TITLE
[1849] Unable to perform stock action. Blocked by Lay/Upgrade track

### ISSUE THREAD

**Issue by Dchoberka** [tobymao/18xx#11348]
https://18xx.games/game/184505

It's my turn in the stock round. Any action that I do produces the error message "Blocking step lay/upgrade track..." that you see in the attached screenshot, and then the action is rolled back.

Interestingly, this also happened yesterday, and it seemed to stop after I sent a chat to the other players about it. That chat never showed up, but I was then able to do a stock action. I tried the same thing today, sending a chat, but it didn't change anything.

Using Firefox on a MAC running Ventura 13.7.1.

<img width="1470" alt="Screenshot 2024-11-14 at 10 15 38 AM" src="https://github.com/user-attachments/assets/c6870b73-3516-427b-9e17-7a8261714ae7">

**Comment by Dchoberka** [comment]
Strangely, I went back after a while and tried it again, and it worked. I have no idea what the cause could have been or why the behavior changed.

**Comment by Dchoberka** [comment]
And now the same error message has returned and I’m unable to do a stock action. What sets the bug off, and what clears it up, is a mystery to me.

**Comment by Dchoberka** [comment]
And I tried again just now, and it worked. Mysterious

**Comment by philcampeau** [comment]
duplicate of  #11138
